src/lib/routes/admin-api/instance-admin.ts

```diff
@@ -35,6 +38,17 @@ class InstanceAdminController extends Controller {
             path: '/statistics/csv',
             handler: this.getStatisticsCSV,
             permission: NONE,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['Instance Admin'],
+                    operationId: 'getInstanceAdminStatsCsv',
+                    responses: {
+                        200: createCsvResponseSchema(
+                            'instanceAdminStatsSchema',
+                        ),
+                    },
+                }),
```

Comment:
This should have a summary and a description.

Reply:
Indeed. Oversight from my side.